src/Text.js

```diff
@@ -27,28 +27,26 @@ class Text extends Entity {
     }
 
     toDxfRows() {
-        const rows = [
+        let rows = [
             new Row('1', this.value),
             new Row('40', this.height),
-            new Row('50', this.rotation), // DEG
-            new Row('10', this.x1), // X
-            new Row('20', this.y1), // Y
-            new Row('30', 0), // Z
-            // 7 --> Text style name (optional, default = STANDARD)
-            // They say Optional but essential for QCad to render the text correctly
+            new Row('50', this.rotation),   // DEG
+            new Row('10', this.x),          // X
+            new Row('20', this.y),          // Y
+            new Row('30', 0),               // Z
             new Row('7', 'STANDARD'),
-            new Row('100', 'AcDbText'),
-        ]
+            new Row('100', 'AcDbText')
+        ];
 
         if (H_ALIGN_CODES.includes(this.hAlign, 1) || V_ALIGN_CODES.includes(this.vAlign, 1)) {
-            rows.push(new Row('11', this.x1)), // X
```

Comment:
Here we still have x1 and y1. Should be x and y

Reply:
Yes my bad